packages/@aws-cdk/aws-ecs-patterns/lib/fargate/application-load-balanced-fargate-service.ts

```diff
@@ -66,6 +66,13 @@ export interface ApplicationLoadBalancedFargateServiceProps extends ApplicationL
    */
   readonly assignPublicIp?: boolean;
 
+  /**
+   * The subnets to associate with the service.
+   *
+   * @default - Public subnets if `assignPublicIp` is set, otherwise the first available one of Private, Isolated, Public, in that order.
+   */
+  readonly vpcSubnets?: SubnetSelection;
```

Comment:
I think I would prefer `taskSubnets` to distinguish this selection from the subnets selected for the load balancer.

Reply:
done!